server/src/main/java/io/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -233,6 +233,35 @@ public boolean isLeader()
 
   public CountingMap<String> getSegmentAvailability()
   {
+    return getSegmentStats(
+        new Function<DataSegment, Integer>()
+        {
+          @Override
+          public Integer apply(DataSegment input)
+          {
+            int available = (segmentReplicantLookup.getTotalReplicants(input.getIdentifier()) == 0) ? 0 : 1;
```

Comment:
If I'm reading this correctly, you have a boolean translating to a binary boolean, then negating it in the return, which is confusing. Can you just take the boolean and return 0 if true and 1 if false?